### PR TITLE
Team 3 bug fixes & easy parameter settings for sim

### PR DESCRIPTION
**Pull request by kunjzk**
# Summary

Easy to set params for simulation to switch between our three behaviours.
Couldn't pick a specific name for an evilPerson, spoilt for choice ;)

## Additional Information

Auxiliary information goes here.

## Test Plan

- Add information on how you tested your changes.

### CODE REVIEW

**Review by Neelesh99 — APPROVED** [review]
LGTM, happy to wait for someone to give a quick lookover before merge

File: internal/clients/team3/clientforaging.go
Comment (lhl2617) [inline internal/clients/team3/clientforaging.go]: decay can be 0

Comment (lhl2617) [reply internal/clients/team3/clientforaging.go]: https://play.golang.org/p/53O2kIyIQl-

**Review by lhl2617 — CHANGES_REQUESTED** [review]
https://play.golang.org/p/53O2kIyIQl-

**Review by lhl2617 — APPROVED** [review]
image

File: internal/clients/team3/clientforaging.go
Comment (Neelesh99) [reply internal/clients/team3/clientforaging.go]: Hold up, this isn't a div by 0 - Victor 2021

Comment (victorbadoual) [reply internal/clients/team3/clientforaging.go]: decay can be 0 indeed https://play.golang.org/p/JcEQbbhBTjU. 
Here is a quick maths problem for you @lhl2617 ;)
merlin_158743359_ff291f8a-d473-4849-9d81-9762826b55f4-superJumbo

Comment (lhl2617) [reply internal/clients/team3/clientforaging.go]: 69

image

Comment (lhl2617) [reply internal/clients/team3/clientforaging.go]: too many params man soz my eyes failed me

**Pull request merged**
